### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n, @zendesk/mintegrations  

### Description

Update en.yml file with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

**Comment by injuhwang** [comment]
👍 for the strings

File: translations/en.yml [hunk 1]
```diff
@@ -5,10 +5,52 @@ packages:
 
 parts:
   - translation:
-      key: "txt.apps.shopify_app.app.description"
-      title: "Description text of the App"
+      key: "txt.apps.shopify_app.app.name"
+      title: "Name for App: Shopify"
+      value: "Shopify"
```
Comment (injuhwang) [inline translations/en.yml]: Please add `hidden: true`.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

**Comment by miogalang** [comment]
This is actually now a hidden app. I'll ask someone to do the same for the v2 branch. 😄  This is the same description we should use right @pmgrove ?

**Comment by pmgrove** [comment]
@miogalang, I guess so unless any of the installation instructions have changed for v2?

**Pull request closed without merging**